AppCheck/Sources/Public/AppCheck/GACAppCheck.h

```diff
@@ -48,6 +37,7 @@ NS_SWIFT_NAME(InternalAppCheck)
 - (instancetype)initWithInstanceName:(NSString *)instanceName
                     appCheckProvider:(id<GACAppCheckProvider>)appCheckProvider
                             settings:(id<GACAppCheckSettingsProtocol>)settings
+                       tokenDelegate:(id<GACAppCheckTokenDelegate>)tokenDelegate
```

Comment:
Why isn't there a corresponding change to FirebaseAppCheck?

Reply:
I haven't finished turning `FIRAppCheck` into a shim yet and that's the only class in Firebase App Check that would be impacted by this change. (The refactor became too large when I tried to do this all in one.)